src/components/Account/Login/Login.component.js

```diff
@@ -32,7 +35,8 @@ export class Login extends Component {
 
   state = {
     isLogging: false,
-    loginStatus: {}
+    loginStatus: {},
+    showPassword: false
```

Comment:
nit: `showPassword` sounds like a fn name (an action), would you mind renaming to something more boolean in nature? perhaps `isPasswordVisible`

Reply:
yes I agree